main.tf

```diff
@@ -10,10 +10,6 @@ locals {
 
 terraform {
   required_providers {
-    aws = {
```

Comment:
Why did this get removed?

Reply:
tflint --recursive threw an item that it wasn't in use by the module.

Reply:
the aws work is being done in the vpc and database modules